apps/chat/src/components/Header/Header.tsx

```diff
@@ -79,36 +83,38 @@ const Header = () => {
       )}
       data-qa="header"
     >
-      <Tooltip isTriggerClickable tooltip={t('Conversation list')}>
-        <div
-          className="flex h-full cursor-pointer items-center justify-center border-r border-tertiary px-3 md:px-5"
-          onClick={handleToggleChatbar}
-          data-qa="chat-panel-toggle"
-        >
-          {showChatbar ? (
-            <>
-              <IconX
-                className="text-secondary md:hidden"
-                width={headerIconSize}
-                height={headerIconSize}
-              />
-
-              <MoveLeftIcon
-                className="text-secondary hover:text-accent-secondary max-md:hidden"
+      {enabledFeatures.has(Feature.ConversationsSection) && (
+        <Tooltip isTriggerClickable tooltip={t('Conversation list')}>
+          <div
+            className="flex h-full cursor-pointer items-center justify-center border-r border-tertiary px-3 md:px-5"
+            onClick={handleToggleChatbar}
+            data-qa="chat-panel-toggle"
+          >
+            {showChatbar ? (
+              <>
+                <IconX
+                  className="text-secondary md:hidden"
+                  width={headerIconSize}
+                  height={headerIconSize}
+                />
+
+                <MoveLeftIcon
+                  className="text-secondary hover:text-accent-secondary max-md:hidden"
+                  width={headerIconSize}
+                  height={headerIconSize}
+                />
+              </>
+            ) : (
+              <MoveRightIcon
+                className="text-secondary hover:text-accent-secondary"
                 width={headerIconSize}
                 height={headerIconSize}
               />
-            </>
-          ) : (
-            <MoveRightIcon
-              className="text-secondary hover:text-accent-secondary"
-              width={headerIconSize}
-              height={headerIconSize}
-            />
-          )}
-        </div>
-      </Tooltip>
-      <CreateNewChatMobile iconSize={headerIconSize} />
+            )}
+          </div>
+        </Tooltip>
+      )}
+      {!isIsolatedView && <CreateNewChatMobile iconSize={headerIconSize} />}
```

Comment:
maybe put this button under feature flag and disable it in same manner as conversations section and etc?